crates/mvr-cli/tests/test_data/test_local_dep_to_git/Move.lock

```diff
@@ -0,0 +1,34 @@
+# @generated by Move, please check-in and do not edit manually.
+[move]
+version = 3
+manifest_digest = "77450CAF5CB6CF95D38E61C7F44F5C35EF483F5B232C93A8039C28ECA9AC8BA1"
+deps_digest = "3C4103934B1E040BB6B23F1D610B4EF9F2F1166A50A104EADCF77467C004C600"
+dependencies = [
+  { id = "Sui", name = "Sui" },
+  { id = "token", name = "token" },
+]
+
+[[move.package]]
+id = "MoveStdlib"
+source = { git = "https://github.com/MystenLabs/sui.git", rev = "framework/mainnet", subdir = "crates/sui-framework/packages/move-stdlib" }
+
+[[move.package]]
+id = "Sui"
+source = { git = "https://github.com/MystenLabs/sui.git", rev = "framework/mainnet", subdir = "crates/sui-framework/packages/sui-framework" }
+
```

Comment:
I'd consider trimming these extra dependencies (and maybe the digests?) out of the test case, to just focus on the local dep. Otherwise it's unclear what's being tested.